scanpy/tests/test_highly_variable_genes.py

```diff
@@ -125,45 +159,35 @@ def test_filter_genes_dispersion_compare_to_seurat():
 
 def test_highly_variable_genes_batches():
```

Comment:
Can you add similar batch HVG tests? I might have missed because of the "black" modifications.

Reply:
sure!

Reply:
I added the test in the seurat_v3 test function.